doc/source/getting_started/WorkingWithInferenceData.ipynb

```diff
@@ -13,11 +13,13 @@
   },
```

Comment:
<p>computation happens in the previous cell with the difference, here we are only selecting the difference we are interested in, I would not use the verb "compute"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>